src/files/infrastructure/mappers/DomainFileMapper.ts

```diff
@@ -0,0 +1,31 @@
+import { FilePaginationInfo } from '../../domain/models/FilePaginationInfo'
+import { FileCriteria, FileSortByOption } from '../../domain/models/FileCriteria'
+import { FileOrderCriteria } from '@iqss/dataverse-client-javascript'
+
+export class DomainFileMapper {
+  static toJSPagination(paginationInfo: FilePaginationInfo): { limit?: number; offset?: number } {
+    return {
+      limit: paginationInfo.pageSize,
+      offset: (paginationInfo.page - 1) * paginationInfo.pageSize
+    }
+  }
+
+  static toJSFileOrderCriteria(criteria?: FileCriteria): FileOrderCriteria {
+    switch (criteria?.sortBy) {
```

Comment:
Since the conversion is so direct, I wonder if we can simplify this kind of mappings between js-dataverse and SPA enums.

Reply:
We could do ` return (criteria?.sortBy as unknown as FileOrderCriteria) || FileOrderCriteria.NAME_AZ;` but is not as type safe

We could also do:
```
Object.values(FileOrderCriteria).find((fileOrderCriteriaKey) => {
      return fileOrderCriteriaKey === criteria.sortBy
    })
```
But I think the readability is worst

Let me know what you think

Reply:
I was thinking of something closer to directly using the js-dataverse types, but I understand that it may impact the design / intention of not coupling js-dataverse to the SPA.

I don't want to give it too much importance now, just something to consider from now.